tests/core/groups/test_group.py

```diff
@@ -0,0 +1,32 @@
+from unittest.mock import MagicMock
+
+
+def test_list_groups_env(client, env_fixture, group, module_mocker):
+    mock_client = MagicMock()
+    module_mocker.patch(
+        'dataall.base.aws.cognito.Cognito',
+        return_value=mock_client
+    )
+    mock_client().list_groups.return_value = ['cognitos', 'testadmins']
+    module_mocker.patch(
+        'dataall.core.groups.api.resolvers.ServiceProviderFactory.get_service_provider_instance',
+        return_value=mock_client()
```

Comment:
Why did we return Cognito() in `test_notification_service` and we return mock_client here?
```
    mocker.patch(
        'dataall.modules.notifications.services.ses_email_notification_service.ServiceProviderFactory.get_service_provider_instance',
        return_value=cognito_client()
    )
```